test/support/channel_case.ex

```diff
@@ -26,7 +26,14 @@ defmodule SkateWeb.ChannelCase do
     end
   end
 
-  setup _tags do
```

Comment:
I ran into issues adding tests to `AuthManager`  since the database wasn't sandboxed - modifying here so all tests using `ChannelCase` use the sandbox by default.